src/main/java/org/mule/extension/ftp/internal/connection/FtpFileSystem.java

```diff
@@ -121,12 +121,14 @@ public void disconnect() {
     } catch (FTPConnectionClosedException e) {
       // this is valid and expected if the server closes the connection prematurely as a result of the logout... ignore
     } catch (Exception e) {
-      LOGGER.warn("Exception found trying to logout from ftp at " + toURL(null), e);
+      LOGGER.warn("Exception found trying to logout from ftp at " + toURL(null));
```

Comment:
Can you use the formater? something like
`LOGGER.warn("Exception found trying to logout from ftp at {} ", toURL(null));`